x-pack/elastic-agent/magefile.go

```diff
@@ -307,6 +314,7 @@ func Package() {
 		packages string
 	}{
 		{"darwin/amd64", "darwin-x86_64.tar.gz"},
+		{"darwin/arm64", "darwin-arm64.tar.gz"},
```

Comment:
So does the Agent binary not actually include any of the individual Beat universal binaries? IIUC this is still packing an arm64 and amd64 Beat package within the Agent universal package.

If that's correct then who is intended consumer of the universal packages for the Beats. Will those be made available for download by end-users as part of the official releases?

Reply:
The beats are build for `darwin/arm64` as well as `darwin/universal`. 
Whenever both `darwin/amd64` and `darwin/arm64` are built, a `darwin/universal` is also built and packaged. 
The final `elastic-agent tar.gz` will contain the universal agent and the beats for both `darwin/amd64` and `darwin/arm64`. Thus the agent will install the platform specific beat.